app/javascript/components/lesson-preview/index.jsx

```diff
@@ -1,26 +1,22 @@
 import React, { useState, useEffect } from 'react';
-import {
-  Tab, Tabs, TabList, TabPanel,
-} from 'react-tabs';
+import PropTypes from 'prop-types';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
 import Prism from 'prismjs';
 
 import LessonContentInput from './components/lesson-content-input';
 import LessonContentPreview from './components/lesson-content-preview';
+import ShareButton from './components/share-button';
 import axios from '../../src/js/axiosWithCsrf';
 
-import { generateLink, encodeContent, decodeContent } from '../../src/js/previewShare';
-
-const LessonPreview = () => {
-  const [content, setContent] = useState('');
+const LessonPreview = ({ previewContent }) => {
```

Comment:
Nit: Can we rename `previewContent` to `sharedContent` please? 

It's difficult to discern the difference between `previewContent` and `content` within these components by their names which could lead to some confusion. I think `sharedContent` will make it clearer what the passed in content represents.